inst/string.m

```diff
@@ -148,23 +149,30 @@
           error ('string: cell inputs must be cellstr');
         endif
         this.strs = in;
-        this.tfMissing = false (size (this.strs));
+        tfM = false (size (this.strs));
```

Comment:
I'm not sure I'm a fan of the `tfM` abbreviation. "M" could be short for a lot of things, and I haven't run in to `tfM` as a conventional name for a missing mask. Maybe `tfNan` or `tfNas` or just `tf`? ("Nas" is for "NaS", my own made-up shorthand for "Not-a-String", analagous to NaN and NaT.)

Reply:
`tf` wourd work fine